src/views/ObservationDetail.vue

```diff
@@ -1,147 +1,106 @@
-        <table class="table">
-          <thead>
-            <tr>
-              <th>Request</th>
-              <th>Site</th>
-              <th>Enclosure</th>
-              <th>Telescope</th>
-              <th>Start</th>
-              <th>End</th>
-              <th>State</th>
-              <th>Priority</th>
-              <th>Modified</th>
-              <th>Created</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr>
-              <td>
-                <router-link :to="{ name: 'requestDetail', params: { id: observation.request } }">
-                  {{ observation.request }}
-                </router-link>
-              </td>
-              <td>{{ observation.site }}</td>
-              <td>{{ observation.enclosure }}</td>
-              <td>{{ observation.telescope }}</td>
-              <td>{{ observation.start | formatDate }}</td>
-              <td>{{ observation.end | formatDate }}</td>
-              <td>{{ observation.state }}</td>
-              <td>{{ observation.priority }}</td>
-              <td>{{ observation.modified | formatDate }}</td>
-              <td>{{ observation.created | formatDate }}</td>
-            </tr>
-          </tbody>
-        </table>
-        <table class="table table-striped">
-          <thead>
-            <tr>
-              <th colspan="4" style="border-right: 2px solid black">Config Status</th>
-              <th colspan="6">Summary</th>
-            </tr>
-            <tr>
-              <th>Id</th>
-              <th>Instrument</th>
-              <th>Guider</th>
-              <th style="border-right: 2px solid black">State</th>
-              <th>State</th>
-              <th>Start</th>
-              <th>End</th>
-              <th>Reason</th>
-              <th>Time</th>
-              <th>Events</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr v-for="c_status in observation.configuration_statuses" :key="c_status.id">
-              <td>{{ c_status.id }}</td>
-              <td>{{ c_status.instrument_name }}</td>
-              <td>{{ c_status.guide_camera_name }}</td>
-              <td style="border-right: 2px solid black">{{ c_status.state }}</td>
-              <template v-if="c_status.summary">
-                <td>{{ c_status.summary.state }}</td>
-                <td>{{ c_status.summary.start | formatDate }}</td>
-                <td>{{ c_status.summary.end | formatDate }}</td>
-                <td>{{ c_status.summary.reason }}</td>
-                <td>{{ c_status.summary.time_completed }}</td>
-                <td><a href="#" @click="openEvents(c_status.summary.events)">View</a></td>
-              </template>
-              <template v-else>
-                <td colspan="6">No summary</td>
-              </template>
-            </tr>
-          </tbody>
-        </table>
-      </template>
-    </div>
-  </div>
+  <data-loader :data-loaded="dataLoaded" :data-load-error="dataLoadError" :data-not-found="dataNotFound">
```

Comment:
The changes in this file are because it is now using the new `DataLoader` component as well as the new `getDataMixin`.